src/System.Security.Cryptography.Cng/src/System/Security/Cryptography/CngPropertyOptions.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+
+namespace System.Security.Cryptography
+{
+    /// <summary>
+    ///     Maps to the "dwFlags" parameter of NCryptGetProperty() and NCryptSetProperty().
```

Comment:
If we ever use these to generate MSDN docs, I assume they’ll have to go through a thorough sweep so I’m not too stressed about that now.

(Given that SSC.Cng is a lightweight interop layer and all but demands knowledge of CNG to use properly, I actually do think that noting the correspondence between the managed enum and the ncrypt.h #define is useful info for callers.)